stream-chat-android/src/main/java/com/getstream/sdk/chat/view/dialog/MessageMoreActionDialog.kt

```diff
@@ -55,13 +55,14 @@ internal class MessageMoreActionDialog(
         setCanceledOnTouchOutside(true)
         binding.startThreadButton.isVisible = canThreadOnMessage()
         binding.copyMessageButton.isVisible = canCopyOnMessage()
-        if (isMessageCreatedByCurrentUser()) {
+        if (isMessageNotCreatedByCurrentUser()) {
```

Comment:
Really interesting to see that this was the other way around 😄

Reply:
🙉  This was quite confusing!